acrarium/Dockerfile

```diff
@@ -13,4 +13,11 @@ RUN true
 COPY --from=builder application/snapshot-dependencies/ ./
 RUN true
 COPY --from=builder application/application/ ./
+
+RUN uid=$(($(($((`date +%s` / 60)) % 165535)) + 100000)) \
```

Comment:
this seems unecessarily complex. Why not a fixed uid?

Reply:
because using a a random id makes it harder for someone to say run a different docker container connected to the same volume as the one you are running and have access to your files, especially if backed by NFS or something similar. Not the end all be all in security but it does increase the barrier to entry for malicious users.